### PR TITLE
pod Affinity

### PR DESCRIPTION
**Pull request by khushimalhoz**
pod affinity

### CODE REVIEW

File: charts/microservice/values.yaml [hunk 1]
```diff
@@ -112,12 +112,29 @@ deployment:
 
   tolerations: []
 
-  affinity: {}
+  affinity:
```
Comment (ashwani-opstree) [inline charts/microservice/values.yaml:115]: Let's make this setting as default in the deployment helm template. If someone wants to override the default setting, he can use these parameters to override.

File: charts/microservice/examples/deploy-nginx.yaml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 global:
-  namespace: "demo-dev"
```
Comment (ashwani-opstree) [inline charts/microservice/examples/deploy-nginx.yaml:2]: Do not update this file. Please create new as example values yaml how we can use the affinity and topoloty

**Comment by ashwani-opstree** [comment]
Please check why these pipeline checks are failing

File: charts/microservice/values.yaml (hunk 1 above)
Comment (khushimalhoz) [reply charts/microservice/values.yaml:115]: I have done the changes 
To make the affinity and tolerations 
in deployment.yaml
 affinity:
        {{- toYaml (merge .Values.deployment.affinity (default dict .Values.deployment.affinity)) | nindent 8 }}
      tolerations:
        {{- toYaml (merge .Values.deployment.tolerations (default dict .Values.deployment.tolerations)) | nindent 8 }}
        
affinity and tolerations settings have default values defined in values.yaml, and these settings can be overridden by user-specified values. The merge function combines the defaults with the user-provided values, ensuring that the user can override the default settings if needed.

File: charts/microservice/values.yaml
Comment (khushimalhoz) [inline charts/microservice/values.yaml:1]: Regarding the checks which are failing 
image
I cannot add DOC to this repo.

File: charts/microservice/examples/deploy-nginx.yaml (hunk 2 above)
Comment (khushimalhoz) [reply charts/microservice/examples/deploy-nginx.yaml]: Have created a affinity.yaml file in examples directory.

**Comment by khushimalhoz** [comment]
> Please check why these pipeline checks are failing
Regarding the checks which are failing 
image
I cannot add DOC to this repo.

**Pull request closed without merging**